lua/sniprun.lua

```diff
@@ -31,9 +31,19 @@ M.config_values = {
     -- "Terminal"
     },
 
-  inline_messages = 0
+  inline_messages = 0,
+
+  -- default highlight stuff goes here
+  SniprunColors={
+    SniprunVirtualTextOk   =  {bg="#66eeff",fg="#000000"},
```

Comment:
I'd like either that gui and cterm colors are configurable, or that the mention that termguicolor is a required option appears somewhere

Reply:
ok. I will add options for cterm .